fibonacci.py

```diff
@@ -0,0 +1,12 @@
+def fibonacci(n):
+    number_fibo = (n+1) * [None]
+
+    def recursive_fibonacci(x):

```

Comment:
do you really need nested functions for this?

Reply:
come to think of it, there's no need for it.